bucoffea/config/vbfhinv.yaml

```diff
@@ -243,6 +241,7 @@ default:
       jet_images: False
       tree_regions:
         - sr_vbf_no_veto_all
+        - sr_vbf
```

Comment:
Saving a tree can be expensive in terms of output size, do we want to keep trees for both regions? I think at least for MC this will be not super useful, though for data we'll need it if we want to analyze things like the ROC curve.

Reply:
done